nixos/modules/services/backup/burpServer.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, networking, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.burpServer;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.burpServer = {
+
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable burp.";
+      };
+
+      user = mkOption {
+        default = "burp";
+        description = "The user to run burp as";
+      };
+
+      group = mkOption {
+        default = cfg.user;
+        description = "The group to run burp as (default same as user)";
+      };
+
+      configFile = mkOption {
```

Comment:
Fair enough. I don't suppose it's possible to name keyfiles or something instead of embedding the passphrase in the configuration file?